src/Admin/FormCaptureAdmin.php

```diff
@@ -37,26 +37,32 @@ class FormCaptureAdmin extends ModelAdmin
 
     private static $menu_icon_class = 'font-icon-checklist';
 
-    public function getEditForm($id = null, $fields = null)
-	{
-		$form = parent::getEditForm();
+    public function getGridField(): GridField
+    {
+        $field = parent::getGridField();
+        $field->setName('Submissions');
 
-		/** @var GridField $gridField */
-		$gridField = $form->Fields()->dataFieldByName($this->sanitiseClassName($this->modelClass));
-		$gridField->setName('Submissions');
+        $this->extend('updateGridField', $field);
```

Comment:
I think `parent::getGridField();` will have already called this hook once? So perhaps it should be something like:

```php
$this->beforeExtending('updateGridField', function (GridField $field) {
    $field->setName('Submissions');
});

return parent::getGridField();
```